variablesview.py

```diff
@@ -289,8 +294,37 @@ def setVariables(self, variables):
 
     def on_item_double_click(self, index):
         name = index.data(Role_Name)
-        self.object_picked.emit(name)
+        parent = index.data(Role_Parent)
+        if parent.name is not None and parent.name is not "":
+            name = self.parent_item_is_container(name, parent)
```

Comment:
I think something like:

```suggestion
            name = self.format_item_name_for_container_access(name, parent)
```

would be easier to immediately understand -- the current function name had me expecting that it would return True or False